index.html

```diff
@@ -1,54 +1,121 @@
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,900;1,400&display=swap" rel="stylesheet">
+  <title>Fern's Resume</title>
+</head>
+
+<body>
+  <h2>Fern Hassan</h2>
+  <h3>
+    Graphic Designer and Illistrator
+  </h3>
+  <div class="contact">
+  <h4>
+    Email: fern.hassan@gmail.com
+  </h4>
+  <h4>
+    Phone: 301-747-8766
+  </h4>
+  <h4>
+    <a href=https://www.linkedin.com/feed/> LinkedIn</a> </h4> 
+  </div>
+<div> 
+<h4>Education</h4>
+  <ul>
+    <li>Maryland Institute College of Art
+      Graphic Design BFA in Progress
+      Expected June 2021
+    </li>
+    <li>Montgomery College
+      Graphic Design AFA, May 2019
+    </li>
+  </ul>
+
+  <h4>Experience</h4>
   <ul>
-    <li>Graphic Designer for Primsa
-January 2020— Current
-One of the founding members of a new graphic design co-op. Currently, we are working on setting up our full lunch in May, and brand packing for Annapolis Pride 2020. </li>
-    
-     
+    <li><strong>Graphic Designer for Primsa January 2020—Current </strong> One of the founding rrrmembers of a new
+      graphic
+      design co-op. Currently, we are working on setting up our full lunch in May, and brand packing for Annapolis
+      Pride 2020. </li>
+    <li> <strong> Freelance Designer
+        2016— Current </strong>
+      Worked as both a freelance volunteer graphic designer and for pay since highschool. Have worked with local
+      businesses and created signage for them. Also took on many posters and informational flyer designs throughout my
+      college career.
+    </li>
+    <li><strong> Third-Key at Claires
+        May 2019— August 2019 </strong>
+      Responsible for store maintenance and organizing based on corporate direction. Also in charge of leading the other
+      members of the team and customer support.
+    </li>
+  </div> 
+  <div class="table" >
   </ul>
-  
-    <div id="footer">
-        <a href="mica-web.github.io/gd-431/02/debugging-html"> Instructions </a> for this activity.frfnlrn
-    </div>
+  <table>
+    <tr>
+      <th class="list">Technical </th>
+      <th class="list">Other</th>
+    </tr>
+    <tr>
+      <td> Illustrator </td>
+      <td>Brand Strategy</td>
+    </tr>
+    <tr>
+      <td>Indesign</td>
+      <td>Fashion Styling </td>
+    </tr>
+    <tr>
+      <td>Photoshop </td>
+      <td>Digital and print </td>
+
+    </tr>
+
+    <tr>
+      <td>After Effect</td>
+      <td>Motion graphics</td>
+
+
+    </tr>
+    <tr>
+      <td>macOS</td>
+      <td>Digital photography</td>
+
+    </tr>
+    <tr>
+      <td>Windows 10</td>
+      <td>Video Editing</td>
+
+    </tr>
+    <tr>
+      <td>Basic HTML/CSS</td>
+      <td>Screenprinting </td>
+
+    </tr>
+    <tr>
+      <td>Social Network Management</td>
+      <td>Illustration</td>
+
+    </tr>
+    <tr>
+      <td>Microsoft Office/Google Drive</td>
+      <td>Painting</td>
+    </tr>
+    <tr>
+      <td>Github</td>
+    </tr>
+    <tr>
+      <td>Slack </td>
+    </tr>
```

Comment:
Tables require you to _either_ have cells for every heading or to use the `colspan` attribute to override this expectations. Here, it's not really appropriate to use `colspan`, so I'd recommend adding blank cells:

```suggestion
    <tr>
      <td>Github</td>
      <td></td>
    </tr>
    <tr>
      <td>Slack </td>
      <td></td>
    </tr>
```